avaframe/com1DFAPy/com1DFAtools.py

```diff
@@ -603,16 +780,21 @@ def computeForceVect(cfg, particles, dem, Ment, Cres):
 
 
 def computeForceSPH(cfg, particles, force, dem):
+    """ Compute SPH """
+
+    # Load required parameters
     rho = cfg.getfloat('rho')
     gravAcc = cfg.getfloat('gravAcc')
     Npart = particles['Npart']
     nrows = dem['header'].nrows
     ncols = dem['header'].ncols
     csz = dem['header'].cellsize
-    # initialize
-    forceSPHX = force['forceSPHX']
-    forceSPHY = force['forceSPHY']
-    forceSPHZ = force['forceSPHZ']
+
+    # initialize fields for force
+    forceSPHX = np.zeros(Npart)
+    forceSPHY = np.zeros(Npart)
+    forceSPHZ = np.zeros(Npart)
```

Comment:
I was initializating the sph force in the compute force part to be able to deactivate the sph and still have the forceSPH (=0) available

Reply:
so do we still need this?